Makefile

```diff
@@ -0,0 +1,44 @@
+# Define the dependencies
+DEPENDENCIES = chafa convert jq curl
+
+# Define the installation directories
+INSTALL_DIR = /usr/local
+
+# Installation instructions
+install: check_dependencies
+	@echo "\033[1;32m Installing... \033[0m"
+	sudo mkdir -pv $(INSTALL_DIR)
+	sudo cp -rv bin $(INSTALL_DIR)
+	sudo cp -rv share $(INSTALL_DIR)
+	@echo "Installation complete."
+	@echo "updating your $PATH variable"
+
+	@echo -e "\033[1;32m DONE \033[0m pokeshell is now installed"
+	@echo -e "\033[1;32m RELOAD YOUR SHELL BEFORE USING POKESHELL \033[0m"
```

Comment:
This script is not updating the $PATH as far as I can tell, please remove line 14. Since it is not updating the $PATH, reloading will not do anything. pokeshell will work as long as `$(INSTALL_DIR)/bin` is on path.